drape/gl_includes.hpp

```diff
@@ -21,6 +21,19 @@
   #include <GLES2/gl2ext.h>
   #include "android/jni/com/mapswithme/opengl/gl3stub.h"
   #include <EGL/egl.h>
+#elif defined(OMIM_OS_LINUX)
+  #define GL_GLEXT_PROTOTYPES
+  #include <GL/gl.h>
+  #include <GL/glext.h>
+  #include <EGL/egl.h>
```

Comment:
C++17 supports this:
```
#if __has_include(<EGL/egl.h>)
...
#endif
```
We can use it for autodetection of the installed GL version for all platforms.

Reply:
And how would you go about ensuring that the code in this block doesn't suddenly turn into completely dead code, in case the CI environment stops providing that header?

Reply:
`#error` ?

Reply:
But [here](https://github.com/organicmaps/organicmaps/pull/2947#discussion_r918371048) you said that the code should build for contributors even without EGL, so it can't be an `#error`.

Reply:
The idea is to build with
- EGL if the header/libs are there
- GL if the header/libs are there

Or fail with a clear message if there are no GL headers/libs installed

Reply:
I have made EGL conditional.